src/routes/AuthRoutes.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { RouteProps } from "react-router-dom";
+import Login from "../Components/Login";
+import auth from "../services/Auth";
+import { routesWithRedirect } from "./RoutesHelper";
+
+const authRoutes: RouteProps[] = [
+  { path: "/login", element: <Login /> },
+  // { path: "/login-teste", element: <Login /> },
```

Comment:
Deixei esse comentário aqui só pra ajudar a pessoa que for fazer as próximas rotas a entender a estrutura de dados. Se acharem melhor, posso tirar.